meta-security-smack/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -10,6 +10,8 @@ System System::Run rwxat-
 System System::Shared rwxat-
 System ^ rwxa--
 System User rwx---
+System User::Home rwxa--
+System User::App:Shared rwxa--
```

Comment:
The transmutation is activated for these 2 new rules in the last version of Tizen.
After thinking about it during the night I finally think that it is probably better to activate the transmutation.
Thus I will abandon the pull request and submit a new version soon.